src/runtime_src/core/include/ert.h

```diff
@@ -124,7 +124,10 @@ struct ert_start_kernel_cmd {
   };
 
   /* payload */
-  uint32_t cu_mask;          /* mandatory cu mask */
+  union {
+    uint32_t cu_mask;          /* mandatory cu mask */
+    uint32_t return_code;      /* return code from soft kernel*/
```

Comment:
I suggest using 'int' or 'int32_t" (if you prefer) instead of uint32_t

Reply:
done